cypress/objects/loginPage.js

```diff
@@ -0,0 +1,34 @@
+class LoginPage{
+
+
+    visit() {
+        cy.visit('/') 
+    }
+
+    getValidationError() {
+        return cy.get('.Select.not-valid')
+    }
+
+    submit(){
```

Comment:
It is often beneficial to add 'button' so that you know you're talking about a button, as in "submitButton()" :)